Source/WebCore/platform/MIMETypeRegistry.cpp

```diff
@@ -93,7 +93,7 @@ constexpr ComparableCaseFoldingASCIILiteral supportedImageMIMETypeArray[] = {
 #if USE(CG) || ENABLE(APNG)
     "image/apng",
 #endif
-#if USE(AVIF)
+#if HAVE(AVIF) || USE(AVIF)
```

Comment:
uh oh. Why is there both `HAVE(AVIF)` and `USE(AVIF)`? Can we consolidate them?

Reply:
HAVE(AVIF) is used by Apple ports where the system frameworks provide the needed support to render these images.
USE(AVIF) is used by non Apple ports. It is defined in Source/cmake/OptionsWPE.cmake and Source/cmake/OptionsGTK.cmake. They use libavif to handle the AVIF images. If we want to merge HAVE(AVIF) and USE(AVIF) and merge HAVE(WEBP) and USE(WEBP), I think it is better to handle this in a separate patch.

Reply:
Can we make sure we have a bug open for the separate patch?

Reply:
https://bugs.webkit.org/show_bug.cgi?id=241997